plugins/woocommerce/i18n/states.php

```diff
@@ -15,6 +15,20 @@
 
 return array(
 	'AF' => array(),
+	'AL' => array( // Albania states.
+		'AL-01' => __( 'Berat', 'woocommerce' ),
+		'AL-02' => __( 'Durrës', 'woocommerce' ),
+		'AL-03' => __( 'Elbasan', 'woocommerce' ),
+		'AL-04' => __( 'Fier', 'woocommerce' ),
+		'AL-05' => __( 'Gjirokastra', 'woocommerce' ),
+		'AL-06' => __( 'Korça', 'woocommerce' ),
```

Comment:
This should be [Korçë](https://github.com/unicode-org/cldr/blob/release-40/common/subdivisions/en.xml#L88).